examples/admanager_axis/pom.xml

```diff
@@ -118,14 +118,9 @@
 
     <!-- Configuration to use log4j for logging -->
     <dependency>
-      <groupId>log4j</groupId>
-      <artifactId>log4j</artifactId>
-      <version>1.2.16</version>
```

Comment:
I think we don't need to update log4j v1: https://www.slf4j.org/log4shell.html

Reply:
There are other dependabot alerts related to v1, so I thought we might as well upgrade, but LMK if you see any issues with that.